sdk/servicebus/azure_messaging_servicebus/src/error.rs

```diff
@@ -74,7 +74,7 @@ impl ServiceBusError {
     pub fn with_source(
```

Comment:
Same question as before: Are customers expected to create their own `ServiceBusError` instances? If not, the `with_source` should probably be private.